src/foam/dao/PredicatedClientDAO.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2025 Google Inc. All Rights Reserved.
```

Comment:
Wrong copyright.

Reply:
Isn't this the FOAM copyright?

Reply:
/**
 * @license
 * Copyright 2025 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */
Old copyright stay the same but this is for new ones.